### PR TITLE
Reject capture

### LINKED ISSUE #1046: Verify: Cannot reject a capture
## Steps to Reproduce
1. Open _Verify_
2. Select a capture
3. Select _REJECT_ from the side drawer
4. Submit

### Current Behaviour
Rejection fails due to an incorrect URL root in the API call: `https://dev-admin.treetracker.org/undefined/raw-captures/:id/reject`

### Desired Behaviour
Rejection succeeds with a call to `https://dev-k8s.treetracker.org/field-data/raw-captures/:id/reject`
Note that the `field-data` URL is defined as the env var `REACT_APP_FIELD_DATA_API_ROOT`.

### PR DESCRIPTION
**Pull request by SuspenseFallback**
## Description

Captures can now be rejected.

**Issue(s) addressed**

- Resolves #1046 

**What kind of change(s) does this PR introduce?**

- [ ] Enhancement
- [x] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

Rejecting a capture gives `ERROR_404` due to `.env` variable mismatch.

**What is the new behavior?**

Rejecting a capture successfully happens.

## Breaking change

**Does this PR introduce a breaking change?**

No.

### CODE REVIEW

**Comment by SuspenseFallback** [comment]
Due to git errors, I have to close this pull request. I will redo this.

**Comment by SuspenseFallback** [comment]
Sorry for the mistake. I was trying to close another pull request

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
+REACT_APP_WEBMAP_DOMAIN=https://map.treetracker.org
```
Comment (nmcharlton) [inline .env:1]: Looks like you've created this branch from another dev branch rather than `master` and therefore have carried across some commits that aren't related to this fix.

File: src/api/treeTrackerApi.js [hunk 2]
```diff
@@ -120,6 +121,7 @@ export default {
   async rejectCaptureImage(capture, rejectionReason) {
     try {
       log.debug('reject capture', capture.id, rejectionReason);
+      console.log('env url', FIELD_DATA_API);
```
Comment (nmcharlton) [inline src/api/treeTrackerApi.js:124]: Please don't leave `console.log()` statements in the code. If you believe they are useful for debug, use `log.debug()` instead, as it is switched off by default in production but can be enabled if need be.

File: src/components/GrowerDetail.js [hunk 3]
```diff
@@ -193,6 +193,7 @@ const GrowerDetail = ({ open, growerId, onClose }) => {
           setErrorMessage(match.message);
         }
 
+        console.log('grower', match);
```
Comment (nmcharlton) [inline src/components/GrowerDetail.js:196]: Same comment regarding `console.log()` – use `log.debug()` instead.

Comment (nmcharlton) [reply src/components/GrowerDetail.js:196]: Also, I'm not sure what this logging has to do with the fix.

File: src/components/SidePanel.js [hunk 4]
```diff
@@ -1,24 +1,25 @@
-import React, { useState, useContext } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
 import {
   Box,
+  Button,
```
Comment (nmcharlton) [inline src/components/SidePanel.js:3]: I don't mind you committing linter changes like this if there are functional changes to a file, but in this case it's just the ordering of `import` statements that has changed. That tends to cause merge headaches for other developers.

File: src/components/Verify.js [hunk 5]
```diff
@@ -1,42 +1,42 @@
-import React, { useEffect, useState, useContext, useRef } from 'react';
```
Comment (nmcharlton) [inline src/components/Verify.js:1]: Same comment about committing linter-only changes for a file as above.

File: src/context/VerifyContext.js [hunk 6]
```diff
@@ -1,9 +1,11 @@
-import React, { useContext, useState, useEffect, createContext } from 'react';
```
Comment (nmcharlton) [inline src/context/VerifyContext.js:1]: More unnecessary changes.

**Review by nmcharlton — CHANGES_REQUESTED** [review]
Only one line in one file has actually been changed to fix this bug, so there shouldn't be 9 files in the PR.

File: .env (hunk 1 above)
Comment (SuspenseFallback) [reply .env:1]: Oh that was why I had so many git errors, thank you!

**Comment by SuspenseFallback** [comment]
I am going to close this as I made multiple mistakes in git, I am trying to redo this.
